src/cloud-api-adaptor/podvm/Makefile

```diff
@@ -82,15 +84,21 @@ $(IMAGE_FILE): $(BINARIES) $(FILES) setopts
 	mkdir -p toupload
 	if [ "${SE_BOOT}" = "1" ] && [ "${ARCH}" = "s390x" ]; then \
 		qemu-img create -f qcow2 "se-${IMAGE_FILE}" 100G; \
+		# Temporary workaround for installing cryptsetup on RHEL 9.4 and below s390x base images for enabling se \
+		# Due to issue: https://gitlab.com/qemu-project/qemu/-/issues/2054 \
```

Comment:
with these three line will meet issue https://github.com/confidential-containers/cloud-api-adaptor/pull/1924#issuecomment-2255766374

remove these three line, building is ongoing